mgmtd/mgmt_be_adapter.c

```diff
@@ -249,6 +249,8 @@ static void mgmt_register_client_xpath(enum mgmt_be_client_id id,
 {
 	struct mgmt_be_xpath_map **maps, *map;
 
+	maps = NULL;
+
```

Comment:
are we going to get a report down after line 267, because we could be deref'ing `maps` when it's NULL ?

Reply:
all 4 cases that are available are in the switch

enum mgmt_be_xpath_subscr_type {
        MGMT_BE_XPATH_SUBSCR_TYPE_CFG,
        MGMT_BE_XPATH_SUBSCR_TYPE_OPER,
        MGMT_BE_XPATH_SUBSCR_TYPE_NOTIF,
        MGMT_BE_XPATH_SUBSCR_TYPE_RPC,
};

I don't see a path that would actually cause this problem, but the compiler is complaining about something that it should not be complaining.  I'm just making it happy